system/library/music/albums_services.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<node order="3" type="folder" visible="!Library.HasContent(Music) + Library.HasServices">
```

Comment:
Library.HasServices has been made redundant from 3.7.0, Library.HasContent(xxxx) will take care of plex and emby music (or tvshow/movies) content as well

Reply:
Library.HasServices will always return false from now on